webrtc.html

```diff
@@ -6209,6 +6210,13 @@ <h2>Dictionary <code><a>RTCRtpSendParameters</a></code> Members</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn data-idl><code>priority</code></dfn> of type <span class=
+            "idlMemberType"><a>RTCPriorityType</a></span>, defaulting to
+            <code>"low"</code></dt>
+            <dd>
+              <p>Indicates the priority of this encoding. It is specified in
+              [[!RTCWEB-TRANSPORT]], Section 4.</p>
```

Comment:
It is now clear the bitrate of the entire sender, but it is still not specified how much each individual simulcast layer should occupy. I think this is fine, not imposing too much restrictions on the encoders, or how to prioritize between layers.

Reply:
I believe having the priority on the RTCRtpSendParameters is a good thing so browser can decide how much bandwidth it allocates this sender vs the other ones.

But if we are providing a fine grained control for simulcast, I think we should keep the priority on the encodings too to be able to prioritize between the different layers of the simulcast stream and assign bandwidth, drop layers, or apply different protection schemes. 

For example, Chrome always start dropping the highest layer  if there is not enough 
 bandwidth, while an app could decide to maintain the highest and drop the mid layer instead.

Reply:
I think that if the app wants control over this, then calling it "priority" is the wrong thing to do. Let's design a mechanism for that when we have requirements.

Reply:
Not sure why priority is not the correct mechanism for this. According to RTCWEB-TRANSPORT the priority attribute is used for local prioritization and if bwe is know from the congestion control algorithm, configure each codec and each data channel with a target send rate that is appropriate to its share of the available bandwidth.

Once the target bitrate is set for a sender, same mechanism based on the priority attribute can be used to distribute the available bitrate for the sender within the simulcast streams.

Reply:
I just copied what is in the RTCWEB-TRANSPORT doc already. 

Note that this is not about taking priority on congestion control or bandwidth estimation, it is about how to divide the available bandwidth gathered from the bandwidth estimation between the different senders and withing the different simulcast streams of each sender.